schemas/2019-09-16/Microsoft.HealthcareApis.json

```diff
@@ -0,0 +1,284 @@
+      },
+      "required": [
+        "objectId"
+      ],
+      "description": "An access policy entry."
+    },
+    "ServiceAuthenticationConfigurationInfo": {
+      "type": "object",
+      "properties": {
+        "authority": {
+          "type": "string",
+          "description": "The authority url for the service"
+        },
+        "audience": {
+          "type": "string",
+          "description": "The audience url for the service"
+        },
+        "smartProxyEnabled": {
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "If the SMART on FHIR proxy is enabled"
+        }
+      },
+      "description": "Authentication configuration information"
+    },
+    "ServiceCorsConfigurationInfo": {
+      "type": "object",
+      "properties": {
+        "origins": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The origins to be allowed via CORS."
+        },
+        "headers": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The headers to be allowed via CORS."
+        },
+        "methods": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The methods to be allowed via CORS."
+        },
+        "maxAge": {
+          "oneOf": [
+            {
+              "type": "integer",
+              "minimum": 0,
+              "maximum": 99999
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The max age to be allowed via CORS."
+        },
+        "allowCredentials": {
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
```

Comment:
Following property is missing:
"readOnly": false

Reply:
See above comment on readOnly